crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -1,5 +1,57 @@
 #![deny(missing_docs)]
-//! This module is used to configure the conductor
+//! This module is used to configure the conductor.
+//!
+//! #### Example minimum conductor config:
+//!
+//! ```rust
+//! let yaml = r#"---
+//!
+//! ## Configure the keystore to be used.
+//! keystore:
+//!
+//!   ## use an in-process keystore with default database location.
+//!   type: lair_server_in_proc
+//!
+//! ## Configure an admin websocket interface at a specific port.
+//! admin_interfaces:
+//!   - driver:
+//!       type: websocket
+//!       port: 1234
+//!       allowed_origins: "*"
+//!
+//! ## Configure the network.
+//! network:
+//!
+//!   ## Use the holo-provided default production bootstrap server.
+//!   bootstrap_service: https://bootstrap.holo.host
+//!
+//!   ## [deprecated] Please just include this in your config file for now.
+//!   network_type: quic_bootstrap
+//!
+//!   ## Setup a specific network configuration..
+//!   transport_pool:
+//!     ## use webrtc
```

Comment:
```suggestion
//!     ## Use WebRTC, which is the only option for now.
```

Is this the only option? If not, just accept the capitalisation fixes.

Reply:
It's effectively the only option, so it makes sense to say that here.